supplementary/alica_ros_proxy/include/clock/AlicaRosTimer.h

```diff
@@ -139,7 +139,7 @@ class TimerFactory : public alica::IAlicaTimerFactory
     TimerFactory&& operator=(TimerFactory&&) = delete;
     ~TimerFactory() = default;
 
-    std::unique_ptr<alica::IAlicaTimer> createTimer(TimerCb timerCb, alica::AlicaTime period) override
+    std::unique_ptr<alica::IAlicaTimer> createTimer(TimerCb timerCb, alica::AlicaTime period) const override
```

Comment:
This should not be `const` imo & I'm not sure how it is compiling. The creation of timer here takes a mutable (not const) CallbackQ reference, so the object is changed.

Reply:
ok